src/guides/optimism-bridge-widget/index.md

```diff
@@ -0,0 +1,1877 @@
+---
+title: "Web3 Unleashed: Optimistic rollups and bridging with Optimism"
+hide:
+  - navigation
+---
+# Web3 Unleashed: Optimistic rollups and bridging with Optimism
+
+Written by [Emily Lin](https://twitter.com/_emjlin)
+
+Last updated 11/03/2022
+
+## Overview 
+
+In this episode of Web3 Unleashed, we're chatting with [Annie Ke](https://twitter.com/annieyke) from Optimism about all things L2: optimistic roll-ups, bridging, and Optimism Bedrock! At the end, we'll build a bridge widget that leverages the scripts in the [Truffle Optimism Bridge box](https://trufflesuite.com/boxes/optimism-bridge), to embed bridging into your Optimism dapps!
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/i_yhkHgl4tA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+If you want a deeper dive into the bridge contract interfaces and an overview of L1s, L2s, and bridging, catch our presentation at Devcon!
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/ZpSsFXv7Fwk?start=4292" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+The completed code lives [here](https://github.com/trufflesuite/unleashed_optimism_bridge).
+
+## Bridging basics in the Optimism Bridge Box
+
+In this tutorial, we'll be creating an NFT Marketplace on Optimism and then adding Bridget, an Optimism bridge widget to bridge ETH between Ethereum Goerli and Optimism Goerli to our DApp by building on top of our Optimism bridge
+
+NOTE: To do the bridging for our widget, we will be interacting with the Optimism bridge contract that has already been deployed. Therefore, we will only be interacting with Ethereum Goerli and Optimism Goerli. If we wanted to run all of this locally, there are a lot of additional steps, such as deploying all the Optimism contracts to your local Ethereum and local Optimism and running a local sequencer.
+
+### Download System Requirements
```

Comment:
Wondering if we should have ## Setup header above this given it's listing a bunch of stuff (Truffle, Metamask, etc). My initial read of this made me think Truffle was the only dep / thing to setup.